#!/usr/bin/env python
PACKAGE = "test_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wave_time_frequency",          double_t, 0, "A double parameter", 0.2, 0.1, 5.0)
gen.add("wave_frequency",               double_t, 0, "A double parameter", 1.0, 0.1, 5.0)
gen.add("wave_amplitude",               double_t, 0, "A double parameter", 1.0, 0.0, 5.0)
gen.add("circle_wave_time_frequency",   double_t, 0, "A double parameter", 0.2, 0.1, 5.0)
gen.add("circle_wave_frequency",        double_t, 0, "A double parameter", 0.1, 0.1, 1.0)
gen.add("circle_wave_distance",         double_t, 0, "A double parameter", 1.0, 0.0, 5.0)
gen.add("circle_wave_amplitude",        double_t, 0, "A double parameter", 0.2, 0.0, 0.5)

exit(gen.generate(PACKAGE, "pcl_fig", "pcl_fig"))